src/node.cc

```diff
@@ -1423,7 +1423,8 @@ bool IsExceptionDecorated(Environment* env, Local<Value> er) {
 
 void AppendExceptionLine(Environment* env,
                          Local<Value> er,
-                         Local<Message> message) {
+                         Local<Message> message,
+                         bool handlingFatalError) {
```

Comment:
Style: snake_case, not camelCase for parameters (i.e., `handling_fatal_error`.)

Also, an enum might be nicer because it gives a clear hint what the parameter does at the call site.  With bools, I usually use a local variable to make it obvious:
```c++
const bool handling_fatal_error = true;
AppendExceptionLine(env, err, message, handling_fatal_error);
```